### PR TITLE
feat: expression

### PR DESCRIPTION
**Pull request by roywillemse**
Expression step for everyone to use.

Documentation on how to use the step and test it out is here: 
https://github.com/Betty-Services/NextGen-expression

### CODE REVIEW

**Comment by chrisobdam** [comment]
Anything happening here?

**Comment by chrisobdam** [comment]
Anything happening here?

File: functions/expression/1.0/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import templayed from "./templayed";
+
+const Expression = async ({ expression, variables, outputType, logging }) => {
```
Comment (justin-betty) [inline functions/expression/1.0/index.js:3]: This function doesn't need to be async

**Comment by justin-betty** [comment]
Can you please add tests

**Comment by archan937** [comment]
I have two questions:

1. Do we want to emphasise that this is  temporary? (as we don't want to have `eval` available anymore in the near future?)
2. Should still call it `expression` or `evaluate`?

**Review by jwielinga — APPROVED** [review]

**Pull request merged**